project.clj

```diff
@@ -53,6 +53,10 @@
                               "phantom-advanced" ["phantomjs"
                                                   "src/test/js/runner.js"
                                                   "src/test/html/advanced.html"]}}
+
+  :repl-options {;; The large number of ClojureScript tests is causing long compilation times
```

Comment:
When I start my REPL with "lein repl" it compiles all of the ClojureScript test files for each build(figwheel, simple, and advanced) if the compiled JS doesn't exist already.  I agree that it shouldn't need to do that, at most it should need to compile the Figwheel profile only.